physical/raft/logstore/bolt_store.go

```diff
@@ -2,7 +2,9 @@ package logstore
 
 import (
 	"errors"
+	"time"
 
+	"github.com/armon/go-metrics"
```

Comment:
I wonder if we should use the local metricsutil package instead?  That way we'll automatically get the `cluster` label included in our metrics, along with any other new label we decide to want on all metrics in future.